{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入错误的密码", "status": "passed", "attachments": [{"name": "错误的密码", "source": "dc1974b6-469e-4d7a-93ee-f1f8deea25f7-attachment.attach"}], "start": 1651988352628, "stop": 1651988352628}], "attachments": [{"name": "截图", "source": "db416256-5e48-4567-b175-371d2d1f229d-attachment.png", "type": "image/png"}, {"name": "截图", "source": "cb7279b4-6071-4b73-8dda-008edb4848c4-attachment.png", "type": "image/png"}], "start": 1651988352628, "stop": 1651988354425, "uuid": "d1b6c8f9-9816-424e-bf39-eb253ac5ba2b", "historyId": "e042998cd34bf7d92d802f115102d44e", "testCaseId": "ce17e49339328b1be4c6805d1bcb9a3f", "fullName": "test_two.Test#test_22", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录"}, {"name": "severity", "value": "critical"}, {"name": "epic", "value": "xxx测试"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "17260-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}